packages/components/aside/src/aside.js

```diff
@@ -117,5 +120,29 @@ customElementDefineHelper(
 				}
 			}
 		}
+
+		firstUpdated(changedProperties) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "changedProperties". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)